help/md/clamp.md

```diff
@@ -0,0 +1,44 @@
+## name
+Clamp
+## title
+Clamp a stochastic variable to a fixed/observed value
+## description
+`x.clamp(data)` fixes the value of the stochastic variable `x` to the observation `data`, and marks the variable as corresponding to an observation.
+
+## details
+Once clamped, the value of `x` is thus expected to remain constant, unless `x` is subsequently unclamped – either explicitly with `x.unclamp()`, or implicitly with `x.clamp(different_data)`.
+
+`x.setValue()` evaluates probabilities at a specific value of `x` without fixing the value.
+
+## authors
+
+## see_also
+setValue
+unclamp
+## example
+x ~ dnNormal(1, 1)
+y ~ dnNormal(2, 2)
+z := x * y
+
+# Set the observed value of x
+x.clamp(1)
+
+# Evaluate P(z) at x = X, y = 1
+y.setValue(1)
+z.probability()
+
+# Modify the observed value of x
+x.clamp(2) # equivalent to x.unclamp(); x.clamp(2)
+z.probability()
```

Comment:
These lines don't work from me: on calling `z.probability()`, I get:

```
   Error:	Variable of type 'Real' has no method called 'probability'.  You can use '.methods()' to find available methods.
```

I suppose that makes sense – there is no mechanism for RevBayes to figure out what the PDF is for the product of two normally distributed random variables.

Reply:
Oof... thanks for catching that.  I've updated (and tested!) the example.